elasticsearch/pom.xml

```diff
@@ -29,10 +29,15 @@ LICENSE file.
     <name>ElasticSearch Binding</name>
     <packaging>jar</packaging>
     <properties>
-        <elasticsearch-version>0.19.8</elasticsearch-version>
+        <elasticsearch-version>2.1.1</elasticsearch-version>
     </properties>
     <dependencies>
         <dependency>
+            <groupId>net.java.dev.jna</groupId>
+            <artifactId>jna</artifactId>
+            <version>4.1.0</version>
```

Comment:
Interesting, so the normal client dependency will use jna if it's present but won't bring it in?

Including it is fine by me; a comment here I'm the pom about why we need to expressly include it will help avoid having it refactored out in the future.